Equations/src/main/java/EquationsApplication.java

```diff
@@ -47,6 +51,9 @@ public void run(EquationsConfiguration configuration, Environment environment) t
         final UserDAO userDAO = jdbi.onDemand(UserDAO.class);
         final GameDAO gameDAO = jdbi.onDemand(GameDAO.class);
 
+        // Helper
+        PlayerHelper playerHelper = new PlayerHelper(gameDAO, userDAO);
```

Comment:
Here the player object is created. This would be a good place to instantiate the active game manager so that you can pass  this direct to the AGM.